### ISSUE TITLE
increase font size in client

### ISSUE THREAD

**Issue by aaronshaf** [keybase/client#5797]
... perhaps to something on par with Slack?

Thanks.

**Comment by jpwgarrison** [comment]
Ubuntu, i3, on a macbook here and it looks like 4pt type. I would happily settle for hand editing a text file somewhere.

**Comment by bkidwell** [comment]
Keybase client is apparently ignoring the Display DPI setting in Windows 7, too. It doesn't fit in with everything else running on the desktop. This issue will scare away new users.

Please fix the app so that the normal type (tab names, object names, messages, etc.) are no smaller than whatever "100% size" is in every other app on a system.

**Comment by virtadpt** [comment]
The same thing happens on Linux boxen with very high resolution screens (3200x1800).  Is there any way that you could make the font size changeable on the fly (with control-plus or control-minus)?  Or possibly add the default font size to the configuration options?

**Comment by cjb** [comment]
@virtadpt FWIW changing the desktop scaling factor with gsettings works well.

**Comment by jpwgarrison** [comment]
@cjb but what about those of us without gsettings?

**Comment by cjb** [comment]
@jpwgarrison Dunno. Without it in what sense? Can you install it? What are you using?

**Comment by jpwgarrison** [comment]
I am running i3wm on xubuntu. gsettings is installed, but does not even open under i3 or xfce4. FWIW I do have DPI set at 120 and all other applications I use have readable font sizes. Their fonts set by the window manager or in-application.

**Comment by jpwgarrison** [comment]
@cjb [Yakyak](https://github.com/yakyak/yakyak) is a google hangouts client I use that is electron based and has zoom in/out.

**Comment by Blindpete** [comment]
I've installed on Windows and Linux, but the fonts are unreadable for me as I'm a bit blind it hurts my eyes.

**Comment by cfoster** [comment]
Would be nice If there was an option to increase the font size, fonts are a little too small at the moment on iPhone at least

**Comment by cfoster** [comment]
Also would be nice to change background colour - White can be very bright indeed.

**Comment by Fancellu2** [comment]
Yes, proper themeing would be nice, but at least a "darkcula" type theme would be a start.

Would be nice if chat responded to ctrl-mousewheel like intellij does.

**Comment by lholden** [comment]
Yeah... it would be really good if there was a way to increase at least the front size of the chat area. I have a co-worker whose vision isn't that great and he is having a hard time with the app because the font size is a bit small for him.

**Comment by shyiko** [comment]
For what it's worth, font size can be increased by adding `--force-device-scale-factor=1.25` (adjust value if needed) after `/opt/keybase/Keybase` in `/usr/bin/run_keybase` (e.g. `/opt/keybase/Keybase --force-device-scale-factor=1.25 &>> "$logdir/Keybase.app.log" &`).

**Comment by murven** [comment]
@shyiko 
is this a valid parameter on Windows? When I try to run keybase with that switch I just get the help screen telling me this is not a valid parameter.

**Comment by lordbah** [comment]
The --force-device-scale-factor worked for me. I like 2.

**Comment by michaelvolz** [comment]
The `--force-device-scale-factor` config is awesome. But how can I assure on my windows 10 system keybase will boot with that setting. I tried various things, but nothing seems to work. I have to quit and start a new version with the command line switch appended.

**Comment by mvz** [comment]
@cjb Where in gsettings would you change the scaling factor? Won't it affect other applications?

**Comment by mvz** [comment]
To answer my own questions, the gsettings invocation is:
```
gsettings set org.gnome.desktop.interface scaling-factor 2
```

And no, it won't affect other (non-electron) applications. I have no idea why none of the gnome programs are affected by this.

**Comment by ldionmarcil** [comment]
Note that on some distros, like Arch, `run_keybase` will launch the GUI through systemd, so you need to modify the `keybase.gui.service` user unit file in order to add scaling parameters. This file should be in `/usr/lib/systemd/user/keybase.gui.service`. Modify the line `ExecStart=/usr/bin/env bash -c "/opt/keybase/Keybase […]` and add scaling factors there. Then, reload your unit files with `systemctl --user daemon-reload` and restart the GUI with `systemctl --user restart keybase.gui.service`. Should work then!

**Comment by xkr47** [comment]
@mvz For me, firefox, gkrellm etc scale up to double size after after your suggestion. It does scale up the keybase app too, but firefox is now unusably big :/

**Comment by mvz** [comment]
@xkr47 :crying_cat_face:

**Comment by xkr47** [comment]
I wonder if it's possible to have per-app gsettings.. Tried setting a custom XDG_DATA_DIR (with a keybase-specific directory configured to override the setting) before starting keybase, but that seemed to have no effect.

**Comment by om26er** [comment]
Actually on a very "normal" screen, the fonts are still small. Maybe just read system font size as a starting point ?

**Comment by xkr47** [comment]
Hey, #5825 is now implemented, so maybe this can also be closed?

**Comment by prometx** [comment]
I would like to provide my strong encouragement that this should be addressable within the gui's "Settings"...

**Comment by looterz** [comment]
Can we still not increase the font-size? Should be easily adjustable in settings.

**Comment by radupopa2010** [comment]
Hey you can use CTRL + to increase font size as it's mentioned here 
https://github.com/keybase/client/issues/5825#issuecomment-453687111

**Comment by riccardo-metere** [comment]
In my system (Ubuntu 20.04.1 + Plasma Desktop) I do need to use: `CTRL` + `=` for zooming in and `CTRL` + `-` for zooming out.

**Comment by beemparthiban** [comment]
For macOS, i use `command` + `=` to  zoom and `command` + `-` to reduce the zoom.

**Comment by ayurai** [comment]
I want to increase font size, not the entire interface ... years later and there is not such basic feature in the settings.

**Comment by lsandov1** [comment]
> -force-device-scale-factor=1.25 &>> "$logdir/Keybase.app.log" &

thanks you comment. Worked for me on my fedora 40 box
